### ISSUE TITLE
Lots of dropped syscalls in falco

### ISSUE THREAD

**Issue by dlorenc** [ossf/package-analysis#10]
```
Thu Feb  4 01:00:47 2021: Falco internal: syscall event drop. 37200 system calls dropped in last second.
```

From some cursory googling, it appears the solution here is to recompile falco (or convince the helm chart) to recompile falco with a larger ring buffer.

**Comment by oliverchang** [comment]
I've made some progress here and figured out how to compile falco with the patched ring buffer size.

Trawling through the falco code I *think* the relevant place to change is here (which is different from the place referenced in the issues above since that's for the kernel driver instead of the eBPF probe).

https://github.com/falcosecurity/libs/blob/0acbdb729224fc626f994f5ad836009a6a271479/userspace/libscap/scap_bpf.c#L61

I changed this from 8MiB to 128MiB and kicked off some python and npm analysis jobs and it seems like the number of dropped calls have almost all disappeared. I'm still seeing the occassional:

```
{ "output_fields": { ... }, "priority": "Critical", "rule": "Falco internal: syscall event drop", "output": "Falco internal: syscall event drop. 1 system calls dropped in last second." }`)
```

But this is much better than the 10-100k syscall drops we've been seeing.

I also noticed that the CPU usage of falco got dangerously close to the `500m` limit (and a bit over in some cases), so I bumped it a bit.

I'll keep monitoring this to make sure this actually works and isn't just a coincidence, and I'll upload a PR with a script to build a drop-in Docker image with the patched ring buffer for helm.
